lib/Memory/Memorable.hpp

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include <EEPROMex.h>
+#include <EEPROMVar.h>
+
+const int START = 0;
+const int SIZE = 32;
```

Comment:
This should be pack in some namespace or struct e.g:
```c++
namespace Memory
{
const int WiFi_start = 0;
const int WiFi_end = 64;
const int Game_start = 65;
const int Game_end = 2137;
}
```
---
And size should be transfer to `WiFi` class because it refers to size of string in `WiFi` and not to size of memory.